### PR TITLE
fix: Add nacos cert tests

### PR DESCRIPTION
**Pull request by LaurenceLiZhixin**
Signed-off-by: LaurenceLiZhixin <382673304@qq.com>

# Description

Add alicloud/nacos certificate test.

## Issue reference

https://github.com/dapr/components-contrib/issues/1712

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by yaron2 — APPROVED** [review]
lgtm

**Comment by yaron2** [comment]
Thank you @LaurenceLiZhixin

File: tests/certification/bindings/alicloud/nacos/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Alicloud Nacos Binding certification testing
```
Comment (artursouza) [inline tests/certification/bindings/alicloud/nacos/README.md:1]: We need a more detailed test plan before this component can be certified.

Please, compare with the test plans we created for other components.

**Pull request merged**